### PR TITLE
[PROD3-439] Remove can donate

### PR DESCRIPTION
**Pull request by letbueno**
# Description

- Remove can donate and use new hook donatedToday or has ticket

## Does this pull request close any open issues?

- Put here the issue that must be closed with this PR
- eg: closes [#PROD3-439]()

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Dangerous changes
- [ ] Chore (change that does not effect how the code works, eg: change color)
- [ ] Tests

### CODE REVIEW

File: src/pages/donations/CausesPage/index.tsx [hunk 1]
```diff
@@ -102,8 +90,8 @@ function CausesPage(): JSX.Element {
   }
 
   useEffect(() => {
-    refetchCanDonate();
-  }, [JSON.stringify(currentUser)]);
+    refetchTickets();
```
Comment (thaisribon) [inline src/pages/donations/CausesPage/index.tsx:93]: i think this part is unnecessary because we already have a useEffect inside ticketsContext calling refetch

File: src/pages/donations/CausesPage/index.tsx [hunk 2]
```diff
@@ -152,7 +139,10 @@ function CausesPage(): JSX.Element {
           )}
         </S.TitleContainer>
 
-        <CampaignSection cardId="1" />
+        {!hasTickets && currentLang === "pt-BR" && (
```
Comment (thaisribon) [inline src/pages/donations/CausesPage/index.tsx:142]: why did you add currentLang?

File: src/pages/donations/GiveTicketPage/index.tsx [hunk 3]
```diff
@@ -102,7 +101,6 @@ function GiveTicketPage({ isOnboarding = false }: Props): JSX.Element {
             </S.LogosWrapper>
           </S.Header>
           <S.TextContainer>
-            <S.DefaultImage src={Envelope} />
```
Comment (thaisribon) [inline src/pages/donations/GiveTicketPage/index.tsx:105]: I think you need to update your branch with collect branch

File: src/pages/donations/CausesPage/index.tsx (hunk 2 above)
Comment (letbueno) [reply src/pages/donations/CausesPage/index.tsx:142]: actually that is from staging, i think leod change?

File: src/pages/donations/GiveTicketPage/index.tsx (hunk 3 above)
Comment (letbueno) [reply src/pages/donations/GiveTicketPage/index.tsx:105]: truee

File: src/pages/users/ForYouPage/ForYouMenu/NewsSection/index.tsx [hunk 4]
```diff
@@ -72,16 +70,16 @@ function NewsSection() {
...
       registerAction("for_you_news_tab_view");
     } else {
       logEvent("P16_view");
     }
-  }, [canDonate]);
+  }, [donatedToday]);
 
   function renderPage() {
-    if (!canDonate) {
+    if (!donatedToday) {
```
Comment (thaisribon) [inline src/pages/users/ForYouPage/ForYouMenu/NewsSection/index.tsx:82]: here is only donatedToday without the "!"

**Review by thaisribon — APPROVED** [review]
